src/cdef.rs

```diff
@@ -613,34 +615,24 @@ pub fn cdef_filter_frame<T: Pixel>(
             *x = CDEF_VERY_LARGE;
           }
         } else {
-          let rec_stride = rec.planes[p].cfg.stride;
-          for (x, y) in cdef_row.iter_mut().zip(
-            rec.planes[p].data_origin()
-              [(row - 2) * rec_stride..(row - 1) * rec_stride]
-              .iter(),
-          ) {
+          for (x, y) in cdef_row.iter_mut().zip(rec.planes[p][row - 2].iter())
```

Comment:
The old version looks like it iterated over only the top two lines, but it looks like this iterates over all the lines?

Reply:
The old version iterates over each one line as "range bounded inclusively below and exclusively above (start..end)." as in [doc](https://doc.rust-lang.org/std/ops/struct.Range.html).
New version also iterates over one line because the index of PlaneRegionMut returns rows.